hack/build/bazel-build-builder.sh

```diff
@@ -19,22 +19,37 @@ script_dir="$(cd "$(dirname "$0")" && pwd -P)"
 source "${script_dir}"/common.sh
 source "${script_dir}"/config.sh
 
+if [ "${CDI_CONTAINER_BUILDCMD}" = "buildah" ]; then
+    if [ -e /proc/sys/fs/binfmt_misc/qemu-aarch64 ]; then
+        BUILDAH_PLATFORM_FLAG="--platform linux/amd64,linux/arm64"
+    else
+        if [ "$(uname)" = "Linux" ]; then
+            echo "Expecting qemu-user-static to be installed on the host machine, please install it"
```

Comment:
note that `qemu-user-static` is needed only when using cross compile, so it might be a too strong requirement otherwise?

Reply:
sure, done